### ISSUE TITLE
Website/accessibility: Focus indicator is invisible

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#15264]
### Operating system

Linux

### Joplin version

*

### Current behaviour

1. Open https://joplinapp.org/.
2. Try to navigate using the <kbd>tab</kbd> key.
3. Move focus to a button.
4. Observe that no focus indicator is visible.

Note: Some links are underlined when focused, but others (some button-style links) have no focus indicator.

### Expected behaviour

Per [WCAG SC 2.4.13](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance) and [SC 2.4.7](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible), the element that has focus should be clearly indicated.

**Comment by personalizedrefrigerator** [comment]
This is most likely caused by `outline:none` in the website CSS styles:
https://github.com/laurent22/joplin/blob/25048623ce02324eaa073049ca0c9cd28f83de9e/Assets/WebsiteAssets/css/site.css#L7
